tests/defences/detector/evasion/test_binary_activation_detector.py

```diff
@@ -0,0 +1,176 @@
+# MIT License
```

Comment:
Let's use the opportunity to make the tests framework independent. I'll add a few suggestions below.